source/features/infinite-scroll.tsx

```diff
@@ -27,18 +27,24 @@ const inView = new IntersectionObserver(([{target, isIntersecting}]) => {
 });
 
 function copyFooter(originalFooter: HTMLElement): void {
-	// Copy the footer links to the sidebar to make them more accessible. Also keep a copy in the footer.
-	const footer = originalFooter.cloneNode(true);
+    // Copy the footer links to the sidebar to make them more accessible. Also keep a copy in the footer.
+    const footer = originalFooter.cloneNode(true);
 
-	for (const child of footer.children) {
-		child.classList.remove('pl-lg-4', 'col-xl-3');
-	}
+    for (const child of footer.children) {
+        child.classList.remove('pl-lg-4', 'col-xl-3');
+    }
+
+    // Check if the footer already exists in the sidebar
+    const sidebarFooter = $('[aria-label^="Explore"]').find('.footer');
```

Comment:
We're not using jQuery. $ is a shortcut for document.querySelector

Make sure to run `npm run fix` locally to fix the lint issues and `npm run watch` to make sure TypeScript validates it

Reply:
Sounds good